client/app/react-components/Destination.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Destination(props) {
+  let icon;
+  let name;
```

Comment:
Small stylistic thing. These can optionally be defined on the same line.

`let icon, name`;

Reply:
I tried that! But for some reason eslint hates it.